Documentation/Troubleshooting/krew-plugin.md

```diff
@@ -20,6 +20,70 @@ See the [kubectl-rook-ceph documentation](https://github.com/rook/kubectl-rook-c
     kubectl krew install rook-ceph
   ```
 
+## Cluster Health
```

Comment:
Honestly I'm not sure we want to put much effort into adding context to the krew plugin page in the Rook docs. It wasn't captured well in the discussion for https://github.com/rook/kubectl-rook-ceph/pull/58, but I thought we decided to just keep details of the Krew plugin commands in the Krew docs. The only place we would have in the Rook docs is where we integrate Krew commands on the other Documentation pages, like the troubleshooting guide. I don't think we want to maintain details on this main krew page in the Rook docs since it's really duplicative of the Krew docs.

Reply:
agree it is duplicate, removed.